setup.py

```diff
@@ -40,8 +40,16 @@
                   'cf': ['xarray'],
                   'gradient_search': ['shapely'],
                   'xarray_bilinear': ['xarray', 'dask', 'zarr'],
+                  'odc-geo': ['odc-geo'],
                   'tests': test_requires}
 
+all_extras = []
+for extra_deps in extras_require.values():
+    all_extras.extend(extra_deps)
+extras_require['all'] = list(set(all_extras))
+
+setup_requires = ['numpy>=1.10.0', 'cython']
```

Comment:
This isn't actually used anywhere, plus this kind of thing (setup_requires) should be defined in the pyproject.toml build section.

Reply:
Good point. I have to admit I just copied it to get the different extra install options working.